content/en/blog/2021/proxyless-grpc/index.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "gRPC Proxyless Service Mesh"
+description: Introduction to Istio support for gRPC's proxyless service mesh features.
+publishdate: 2021-08-15
+attribution: "Steven Landow (Google)"
+---
+
+## Background
+
+Recent releases of gRPC have added support for consuming [`xDS`](https://blog.envoyproxy.io/the-universal-data-plane-api-d15cec7a)
+directly. This means you can get many of the benefits of a service mesh, without the overhead of a sidecar proxy. 
+
+Istio 1.11 adds experimental support for basic service discovery, some `VirtualService` based traffic policy, and
+mutual TLS.
+
+## Supported Features
+
+The current implementation of the xDS APIs within gRPC is limited in some areas compared to Envoy. The following
+features should work, although this is not an exhaustive list and other features may have partial functionality:
+
+* Basic service discovery. Your gRPC service can reach other pods and virtual machines registered in the mesh.
+* [`DestinationRule`](/docs/reference/config/networking/destination-rule/):
+  * Subsets: Your gRPC service can split traffic based on label selectors to different groups of instances. 
+  * The only Istio `loadBalancer` currently supported is `ROUND_ROBIN`, `consistentHash` will be added in the future.
+  * `tls` settings are restricted to `DISABLE` or `ISTIO_MUTUAL`. Other modes will be treated as `DISABLE`.
+* [`VirtualService`](/docs/reference/config/networking/virtual-service/):
+  * Header match and URI match in the format `/ServiceName/RPCName`
+  * Override destination host and subset.
+  * Weighted traffic shifting.
+  * // TODO does timeout work (I think we must set MaxStreamDuration for this to work – I don't think we do today)
+  * // TODO do retries work
```

Comment:
They're adding some retry policy support, but partially implemented in C/Java, not in Go.